src/lib.rs

```diff
@@ -2311,6 +2321,10 @@ fn trim_start_matches(s: &str, c: char) -> &str {
 
 /// Helper trait describing values which may be returned by macro implementation
 /// methods used by this crate's macros.
+#[cfg(all(
+    not(all(target_arch = "wasm32", any(target_os = "unknown", target_os = "wasi"))),
+    feature = "proc-macro"
+))]
 pub trait MacroResult {
```

Comment:
Let's keep this trait available with the feature disabled, and provide a default implementation of `fn into_stream(self)` which is provided if the feature is enabled. That way we can keep the `test_derive!` macro available on wasm.

Reply:
I don't get how `fn into_stream(self)` default implementation should look like. Can you elaborate here?

Reply:
I'll do the change in a follow-up commit, it's not a super substantial one :-)